src/MusicalScore/Graphical/VexFlow/CanvasVexFlowBackend.ts

```diff
@@ -106,28 +107,29 @@ export class CanvasVexFlowBackend extends VexFlowBackend {
             this.rules,
             fontFamily
         );
-        this.CanvasRenderingCtx.fillStyle = color;
-        this.CanvasRenderingCtx.strokeStyle = color;
+        if (color) {this.CanvasRenderingCtx.fillStyle = color;}
+        if (color) {this.CanvasRenderingCtx.strokeStyle = color;}
         this.CanvasRenderingCtx.fillText(text, screenPosition.x, screenPosition.y + heightInPixel);
         this.CanvasRenderingCtx.restore();
         this.CanvasRenderingCtx.font = old;
         return undefined; // can't return svg dom node
     }
-    public renderRectangle(rectangle: RectangleF2D, styleId: number, colorHex: string, alpha: number = 1): Node {
+    public renderRectangle(rectangle: RectangleF2D, styleId: number, colorHex: string, alpha: number = 1): Node | undefined {
         const old: string | CanvasGradient | CanvasPattern = this.CanvasRenderingCtx.fillStyle;
         if (colorHex) {
             this.CanvasRenderingCtx.fillStyle = colorHex;
         } else {
-            this.CanvasRenderingCtx.fillStyle = VexFlowConverter.style(styleId);
+            const fillStyle: string | undefined = VexFlowConverter.style(styleId);
+            if (fillStyle) {this.CanvasRenderingCtx.fillStyle = fillStyle;}
         }
         this.CanvasRenderingCtx.globalAlpha = alpha;
         this.ctx.fillRect(rectangle.x, rectangle.y, rectangle.width, rectangle.height);
         this.CanvasRenderingCtx.fillStyle = old;
         this.CanvasRenderingCtx.globalAlpha = 1;
-        return undefined; // can't return dom node like with SVG
+        return undefined; // can't return svg dom node
```

Comment:
please revert this comment change. the previous one is clearer. of course you can't return a svg node in canvas.

Reply:
Only this instance or make the same comment on the other instances within CanvasVexFlowBackend? They should all be the same within this file, should not them?

Reply:
Ok I will unify to the clearer one

Reply:
Done